### ISSUE TITLE
Error use of wavelet.rec  and wavelet.dec?

### ISSUE THREAD

**Issue by wangshuxinxinxin** [LiQiufu/WaveSNet#3]
Is the function wavelet.rec in DWT and wavelet.dec in IDWT reversed?

see https://github.com/fbcotter/pytorch_wavelets/blob/master/pytorch_wavelets/dwt/transform2d.py#L25 and https://github.com/fbcotter/pytorch_wavelets/blob/master/pytorch_wavelets/dwt/transform2d.py#L94

**Comment by LiQiufu** [comment]
Yes. The wavelet.rec and wavelet.dec can be reversed in DWT and IDWT.

**Comment by LiQiufu** [comment]
Yes. The wavelet.rec and wavelet.dec can be reversed in DWT and IDWT.
